docs/styles.rst

```diff
@@ -0,0 +1,128 @@
+Styles
+======
+
+VSG supports several predefined styles.
+They can be used with the **--style** command line option.
+
+The table below lists the built in styles available
+
++---------------+--------------------------------------------+
+| Style         | Description                                |
++---------------+--------------------------------------------+
+| legacy        | Rules based on the pre 2.0.0 release       |
+| lowercase     | All keywords and identifiers are lowercase |
++---------------+--------------------------------------------+
```

Comment:
This is just example right? What I mean is that each style would need separate section. It is not enough to write 1 sentence. The question arises, how we should name styles. lowercase is not good. I have an idea. Maybe you try to describe only legacy style, as I guess this is the one you use. Also probably do not name it legacy, but write in the doc that it is the style that was used as the default one for vsg so far.

Reply:
correct, this is just an example to illustrate the concept.